maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/ServerHelper.java

```diff
@@ -0,0 +1,54 @@
+package org.openmrs.maven.plugins.utility;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ServerHelper {
+    private Wizard wizard;
+    private static final int MAX_USHORT_VALUE = 65535;
+
+    public ServerHelper(Wizard wizard) {
+        this.wizard = wizard;
+    }
+
+    public boolean isPortInUse(int port) {
+        String message = String.format("Checking if port %s is in use...", port);
+        wizard.showMessage(message);
+        return checkPortUsage(port);
+    }
+
+    public boolean checkPortUsage(int port) {
+        ServerSocket serverSocket = null;
+        try {
+            serverSocket = new ServerSocket(port);
+            this.closeTestingServerSocket(serverSocket);
+        } catch (IOException e) {
+            return true;
+        }
+        return false;
+    }
+
+    private void closeTestingServerSocket(ServerSocket serverSocket) throws IOException {
+        serverSocket.close();
+
+        while (!serverSocket.isClosed()) {
+            try {
+                Thread.sleep(500);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```

Comment:
Did you take a look at our conventions regarding the use of exception.printStackTrace? https://wiki.openmrs.org/display/docs/Java+Conventions